lib/rss/rss.rb

```diff
@@ -950,7 +950,7 @@ def set_next_element(tag_name, next_element)
       prefix << "#{klass.required_prefix}_" if klass.required_prefix
       key = "#{prefix}#{tag_name.gsub(/-/, '_')}"
       if self.class.plural_forms.has_key?(key)
-        ary = __send__("#{self.class.plural_forms[key]}")
+        ary = __send__(self.class.plural_forms[key].to_s)
```

Comment:
```suggestion
        ary = __send__("#{self.class.plural_forms[key]}")
```

Not in `Style/StringLiterals`.

Not in `Style/StringLiterals`.